Dockerfile

```diff
@@ -79,6 +86,5 @@ ENV RAILS_ENV=production
 COPY --from=builder $BUNDLE_APP_CONFIG $BUNDLE_APP_CONFIG
 COPY --from=builder --chown=vets-api:vets-api /srv/vets-api/src ./
 COPY --from=builder --chown=vets-api:vets-api /srv/vets-api/clamav/database ../clamav/database
-RUN if [ -d certs-tmp ] ; then cd certs-tmp ; for i in * ; do cp $i /usr/local/share/ca-certificates/${i/pem/crt} ; done ; fi && update-ca-certificates
```

Comment:
certs-tmp is used in https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/build/roles/vets-api/tasks/main.yml#L90 to load in the host system ca's, so this removal will break that.

why not move this line to base stage as-is and have review instances follow the same process as linked above (make a temp dir, copy into it, mount into container during build, whack dir when done" ?

Reply:
Yeah, I know where this gets picked up and have a branch to update it, should we decide to do this.

I want review instances to be setup to run a build again outside of Ansible. So, ideally, I'd like to put them in a directory that's gitignore'd that can hang out indefinitely on those instances. I could just gitignore this particular path, just seemed nice to get it out of the root of the app if it was going to hang out like that.

Reply:
And actually, this won't work in base because we haven't copied in files from the app into the image yet. That only happens in the builder target and beyond.

Reply:
ahh yeah, totally valid.  This looks fine, tough I would ask that you leave this RUN statement in for now and remove it separately once the build side in devops is there.  That should reduce rollout complexity by not requiring coordination across two repos at the cost of some duplication here.

Reply:
Done!